source/fundamentals/crud/write-operations/modify.txt

```diff
@@ -56,40 +56,73 @@ You can call the ``updateOne()`` and ``updateMany()`` methods on a
 
 .. code-block:: java
 
-    collection.updateOne(query, updateDocument);
-
-    collection.updateMany(query, updateDocument);
+    collection.updateOne(<query>, <updateDocument>);
+    collection.updateMany(<query>, <updateDocument>);
     
 Update Operation Parameters
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The ``updateOne()`` and ``updateMany()`` methods both have the following
 parameters:
 
-- ``query`` specifies a query filter with the criteria to match documents to update in your collection
-- ``updateDocument`` specifies the fields and values to modify in the matching document or documents. For this example, we use the :doc:`Updates builder </fundamentals/builders/updates>` to create the update document.
+- ``query`` specifies a query filter with the criteria to match
+  documents to update in your collection.
+- ``update`` specifies the fields and values to modify in the matching
+  document or documents. For this example, we use the :doc:`Updates
```

Comment:
S: Since the style guide suggests to [use "we" sparingly](https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/), I think you can reword in this case:
```suggestion
  document or documents. This example uses the :doc:`Updates
```